crates/bevy_pbr/src/render/pbr_fragment.wgsl

```diff
@@ -79,6 +79,16 @@ fn pbr_input_from_standard_material(
     // Neubelt and Pettineo 2013, "Crafting a Next-gen Material Pipeline for The Order: 1886"
     let NdotV = max(dot(pbr_input.N, pbr_input.V), 0.0001);
 
+    // Unpack `ddx_uv` and `ddy_uv` so that we can call `sample_texture` later.
+    // If this isn't a meshlet shader, just come up with placeholders.
+#ifdef MESHLET_MESH_MATERIAL_PASS
+    let ddx_uv = in.ddx_uv;
+    let ddy_uv = in.ddy_uv;
+#else   // MESHLET_MESH_MATERIAL_PASS
+    let ddx_uv = vec2<f32>(0.0);
```

Comment:
perhaps it would be better if `sample_texture` took a struct who's declaration varied based on the MESHLET define?

not a big deal though

Reply:
Done.